lib/policyEvaluator/utils/checkArnMatch.js

```diff
@@ -35,7 +35,11 @@ function checkArnMatch(policyArn, requestRelativeId, requestArnArr,
         const segmentRegEx = new RegExp(regExofArn[j]);
         const requestSegment = caseSensitive ? requestArnArr[j] :
             requestArnArr[j].toLowerCase();
-        if (!segmentRegEx.test(requestSegment)) {
+        const policyArnArr = policyArn.split(':');
+        // We want to allow an empty account ID for utapi service ARNs
```

Comment:
I'll also add that the ability to omit the account ID is needed for service-level metrics.